.github/workflows/ci.yml

```diff
@@ -14,8 +14,14 @@ jobs:
       - name: Checkout Repository
         uses: actions/checkout@v4
 
+      - name: Setup Go
+        uses: actions/setup-go@v4
+        with:
+          go-version: 1.21.x
+          cache: false
```

Comment:
manual fix: `v3.0.0+` requires explicit setup-go installation step prior to using this action
See: https://github.com/golangci/golangci-lint-action#compatibility